### DISCUSSION TITLE
Will there be support for Labelbox annotations in the future?

### DISCUSSION THREAD

**Discussion by freds-dev** [laclouis5/globox#43] [General]
Hi,

I came across this tool and was wondering if you will support Labelbox annotations in the future?

Thanks!

**Comment by laclouis5** [comment]
Hi,

I took a rapid look at the specifications for the import/export of annotations in Labelbox. While the import from this format could be possible, at first look I doubt that exporting to this format would be simple to do.

The issue seems to be that the LabelBox format is highly coupled to their annotation tools. As such, it stores a lot of other informations such as the project settings, URLs, etc... This information may not be available in other supported annotation formats such as YOLO, thus, other annotations formats cannot be exported to LabelBox.

Could you please send me some Labelbox exports so that I can investigate the feasibility? For instance a small dataset of bounding box annotations with various classes and image sizes. You could send both the labels export (ground truths) and the model run exports (prediction). See [this page](https://docs.labelbox.com/reference/export-v2-glossary#project_details) for more details.
